### ISSUE TITLE
Add AppImage support

### ISSUE THREAD

**Issue by io12** [burtonageo/cargo-bundle#75]
AppImage is another useful format besides rpm and deb.

https://appimage.org/

**Comment by ndarilek** [comment]
I need this too and was thinking of attempting it.

Seems like the other implementations write their formats natively, but AppImage looks a bit more challenging as it's essentially a squashfs/ISO9660 image. Would an implementation that calls out to the official AppImage tool be accepted? I *really* don't want to implement the spec directly if I don't have to. :)

**Comment by mdsteele** [comment]
Yes, that would be a welcome contribution!  It would be great to have support for this.  (It's nice to not need to install additional tools to be able to use cargo-bundle, but support for AppImage that requires installing tools is definitely better than no support at all!)

**Comment by ndarilek** [comment]
Cool, I was just pointed to linuxdeploy as a good tool for automatically 
creating appimages. I don't know much about the format, but I'll try to 
generate packages on my own first, and will then see if there is any 
benefit to integrating the process with `cargo bundle`. At least the 
appimage tooling is almost entirely self-contained and pretty easy to 
install.
